### ISSUE TITLE
Pull request to Spacemacs?

### ISSUE THREAD

**Issue by jluttine** [sdwolf/spacemacs-xclip#1]
Hi, thanks for this! I'm running Spacemacs in terminal (`emacs -nw`) and this layer successfully added support for yanking into system clipboard. So it works. Any plans to make a pull request to Spacemacs?

**Comment by sdwolf** [comment]
I'm not sure it would be a good idea. From that I see, new layers get merged really slowly, and the functionality for this one is trivial. That is the reason why I documented the [A better alternative](https://github.com/sdwolf/spacemacs-xclip#a-better-alternative) section.

When I first made this I used to clone this repo into my private layers directory, but now I just have those lines to my [.spacemacs](https://github.com/sdwolf/dotfiles/blob/96cac68c41ee1e0aaafcbd2bea81506be088c8e9/.spacemacs#L434-L441).

Maybe instead of submitting the layer I will make a PR to document clipboard support for terminals...
